app/2.1.x/hybrid-mode.md

```diff
@@ -88,6 +88,37 @@ Note that Control Plane still needs a database (Postgres or Cassandra) to store
 Data Plane nodes. You may run more than a single Control Plane nodes to provide load balancing
 and redundancy as long as they points to the same backend database.
 
+## PKI mode
+
+Starting in Kong 2.1, the Hybrid cluster can use certificates signed by a central certificate authority (CA).
+This mode can be activated by setting `cluster_mtls` to `"pki"` in `kong.conf`. The default value is `"shared"`.
+
+In PKI mode, the Control Plane and Data Plane don't need to use the same `cluster_key` and `cluster_cert_key`.
+Instead, Kong validates both sides by checking if they are from the same CA. This eliminates the risk of
+transporting private keys around.
+
+Set the following configuration parameters in `kong.conf` on the Control Plane:
+
+```
+cluster_mtls = pki
+cluster_ca_cert = /path/to/ca-cert.crt
+```
+
+`cluster_ca_cert` specifies the root CA certificate for `cluster_cert` and `cluster_cert_key`. This
+certificate must be the root CA certificate and not any of an intermediate CA.
+Kong allows at most `3` levels of intermediate CAs to be used between the root CA and the cluster certificate.
+
+Set the following configuration parameters in `kong.conf` on the Data Plane:
```

Comment:
Maybe we should add the usage of `cluster_cert` and `cluster_cert_key` as well. Same for the CP examples above.

Reply:
👍 added